src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartBuilderPreview.test.tsx

```diff
@@ -184,7 +184,8 @@ describe('ChartBuilderPreview', () => {
     render(
       <ChartBuilderPreview
         chart={produce(testMapChartRenderer, draft => {
-          draft.boundaryLevel = undefined;
+          (draft as Chart & { boundaryLevel?: number }).boundaryLevel =
```

Comment:
This doesn't seem right. We've now made `boundaryLevel` required in `ChartRenderProps`, so the correct way to set this would be simply:

```ts
draft.boundaryLevel = 0;
```